### PR TITLE
deprecate gmcommand builtin

### PR DESCRIPTION
**Pull request by mekolat**
# REQUIRES https://github.com/themanaworld/tmwa/pull/49

### CODE REVIEW

**Comment by wushin** [comment]
Doesn't that set you to BaseLevel 1?

**Comment by mekolat** [comment]
oh oops sorry I didn't pay attention

File: world/map/npc/annuals/xmas/core.txt [hunk 1]
```diff
@@ -159,8 +159,8 @@ L_LevelingReward:
     goto L_XmasExp;
 
 L_XmasLevel:
-    gmcommand "@blvl 1";
-    gmcommand "@l I was a good kid this year.";
+    gmlog "XMAS: increase BaseLevel ("+BaseLevel+" => "+(BaseLevel+1)+")";
+    set BaseLevel, BaseLevel + 1;
```
Comment (wushin) [inline world/map/npc/annuals/xmas/core.txt]: remove this from the gmlog all the GMs thought it was annoying.

Comment (mekolat) [inline world/map/npc/annuals/xmas/core.txt]: @wushin done

**Pull request merged**